app-doc/kicad-doc/kicad-doc-5.0.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Electronic Schematic and PCB design tools manuals"
+HOMEPAGE="http://www.kicad-pcb.org/"
+SRC_URI="https://github.com/KiCad/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( GPL-3+ CC-BY-3.0 ) GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="html +pdf"
+LANGS="ca de en es fr id it ja pl"
+for lang in ${LANGS}; do
```

Comment:
Make `LANGS` an array, and use `${LANGS[@]/#/l10n_}` or alike.

Reply:
Actually even that is not needed. We can directly define the l10n_ flags and use them.